### ISSUE TITLE
Explanation of array keys

### ISSUE THREAD

**Issue by michellesanver** [PHPEmbark/PHPEmbark#14]
Instead of having a key of 1 and one of 2, to really from the beginning make it clear an idea would be to have a key of 1, another of "foo" and another of 5 or so, to show that it's REALLY you defining the keys and that it can be anything.

**Comment by ramsey** [comment]
Agreed. We have a slide on numeric indexes. We should add a slide on "key" indexes (or hashes).
